### PR TITLE
template update: attempt to resolve fails in another template PR

### PR DESCRIPTION
**Pull request by projectoriented**
## PR checklist
Attempting to close https://github.com/nf-core/raredisease/pull/64 with this
- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by projectoriented** [comment]
@jemten @ramprasadn Hello 👋 ! I made this PR to reverse changes I made before the nf-core/tools release- changes such as ignoring unrecognized files in `.nf-core.yml`. Please review so I can hopefully merge the bot template PR 👍

File: .nf-core.yml [hunk 1]
```diff
@@ -2,13 +2,6 @@ lint:
   files_unchanged:
     - .github/CONTRIBUTING.md
     - assets/multiqc_config.yaml
-    - .github/workflows/linting.yml
     - lib/NfcoreTemplate.groovy
     - lib/NfcoreSchema.groovy
     - assets/sendmail_template.txt
```
Comment (ramprasadn) [inline .nf-core.yml:7]: > lib/NfcoreTemplate.groovy
   lib/NfcoreSchema.groovy
   assets/sendmail_template.txt

I added the two groovy files to this section as I had some tests failing, and it looks like the file `sendmail_template.txt` was also added recently by you. But perhaps we don't need them in this list anymore? What do you think?

Comment (projectoriented) [reply .nf-core.yml:7]: Yup! I forgot to remove them :)

**Review by jemten — APPROVED** [review]
👏

**Pull request merged**